### DISCUSSION TITLE
Can't  create new item

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15363] [Archive]
Hi, 

I have a huge problem with one of my collections, name "Stages", where I can't create a new item with error message: "The following fields have invalid values: ID: Value is required" and "id" field is always readonly... 

But by field "id" (standard field) in collection "Stages" checkboxes "Required" and "Readonly" are not ON... Please see screen-video. Is it some Bug?

https://user-images.githubusercontent.com/54798086/188112530-0e5fa1dd-aa3a-49f4-8b88-1777dc6f1d4a.mp4

**Comment by azrikahar** [comment]
Can you try disabling the `Unique` option in the Schema tab for the `id` field?

On that note, did you enable it by yourself beforehand? and what database are you using? It does seems odd how it is set as unique for an autoincrement integer, which by itself should already be unique. I've tested it and it's not on by default for autoincremented primary keys.

**Reply by rynskiy** [reply]
Hi @azrikahar as soon as I tried to uncheck the checkbox, an error appeared (see screenshot). I think I have set a Unique checkbox by creating this collection.
![error](https://user-images.githubusercontent.com/54798086/188188387-fa4d3267-d7f8-4df6-86ac-7e45804017f4.PNG)

**Reply by azrikahar** [reply]
Can you share what database are you using? That'd be helpful! I believe you can try recreating `stages` without setting it as unique to see how that goes, just so we can verify it was accidentally set as unique. 

I think the error is technically correct and this seems to be an uncovered edge case, but I do agree the app should probably have prevented setting unique on autoincremented integers in the first place (unless there's an actual valid use case for it).

Once you verified it works when recreating the collection without setting unique on the autoincremented id, we'll discuss what is the best course of action to prevent future unintentional configs such as this 👍

**Reply by k0l0ssus** [reply]
I've dealt with this intermittently for the same collection. Last time this happened, the reason was that the sequence generator set as the default value on the collection table
